tephra-core/src/main/java/co/cask/tephra/runtime/ZKModule.java

```diff
@@ -52,16 +62,68 @@ private ZKClientService provideZKClientService(Configuration conf) {
       zkStr = conf.get(TxConstants.HBase.ZOOKEEPER_QUORUM);
     }
 
-    return ZKClientServices.delegate(
```

Comment:
Not using `ZKClientServices.delegate` since it returns `org.apache.twill.zookeeper.ZKClientService` instance.